### ISSUE TITLE
Prefer (cell) numbers marked as standard

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#175]
```
If there is more then one cell number for an contact, prefer the one marked as 
standard if possible.


```

Original issue reported on code.google.com by `fschm...@gmail.com` on 21 May 2011 at 12:59

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 24 Jun 2011 at 11:15
- Changed state: **WaitingForRelease**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v3.0
```

Original comment by `Florent....@gmail.com` on 15 Jul 2011 at 3:35
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 25 Aug 2011 at 7:50
- Changed state: **Deployed**
